paulshryock/paul-shryock
# [#46]/[#47]: Serve and watch local development changes

## Summary
During development, add the ability to serve websites locally to test in the browser. Watch for changes and re-run build tasks to avoid the need to re-run full builds.

## Testing
- Run `npm run serve`
- See that all websites are served to `localhost` on different ports beginning with `8000`, each incremented by `1`
- See that file changes are watched and rebuilt to server(s)

## Issue(s)
- Closes #46
- Closes #47

## Changelog

### Added
- Add local development server. [#46]
- Watch for changes and re-run build tasks. [#47]

### Changed
- Moved documentation to Readme. Will split out later if it gets too large.

## Release Checklist
